streaming/base/dataset.py

```diff
@@ -678,6 +682,11 @@ def _resume(self, world: World, epoch: int) -> Tuple[int, int]:
         sample_in_epoch = obj['sample_in_epoch']
         self.num_canonical_nodes = obj['num_canonical_nodes']
         self.shuffle_seed = obj['shuffle_seed']
+        if 'initial_physical_nodes' in obj:
```

Comment:
nit: Does this `obj.get('initial_physical_nodes', None)` works?

Reply:
yes totally. changed it